clients/flash/common-library/src/main/actionscript/org/bigbluebutton/lib/settings/views/chat/ChatSettingsViewMediatorBase.as

```diff
@@ -0,0 +1,9 @@
+package org.bigbluebutton.lib.settings.views.chat {
+	import robotlegs.bender.bundles.mvcs.Mediator;
+	
+	public class ChatSettingsViewMediatorBase extends Mediator {
+		
+		[Inject]
+		public var view:ChatSettingsViewBase;

```

Comment:
This class needs to stay in the repository as it will contain the chat settings view logic.